cypress/e2e/resources_spec/resources_creation.cy.ts

```diff
@@ -0,0 +1,82 @@
+import { PatientDetails } from "@/pageObject/Patients/PatientDetails";
+import { PatientEncounter } from "@/pageObject/Patients/PatientEncounter";
+import { FacilityCreation } from "@/pageObject/facility/FacilityCreation";
+import { ResourcesCreation } from "@/pageObject/resources/ResourcesCreation";
+
+const facilityCreation = new FacilityCreation();
+const patientEncounter = new PatientEncounter();
+const patientDetails = new PatientDetails();
+const resourceCreation = new ResourcesCreation();
+
+describe("Resources Management", () => {
+  const facility = "DH Aluva";
+  const status = "Pending";
+  const category = "Medicines";
+  const resourceTitle = "TestCypress";
+  const reasonOfRequest = "Reason Testing";
+
+  beforeEach(() => {
+    cy.loginByApi("devnurse");
+    cy.visit("/");
+  });
+
+  it("Create a new resource", () => {
+    facilityCreation.selectFacility("GHC Payyanur");
+    patientEncounter
+      .navigateToEncounters()
+      .openFirstEncounterDetails()
+      .clickPatientDetailsButton();
+
+    patientDetails.clickResourcesTab().clickCreateRequestButton();
+
+    resourceCreation
+      .selectFacility(facility)
+      .selectStatus(status)
+      .selectCategory(category)
+      .enterResourceTitle(resourceTitle)
+      .enterReasonOfRequest(reasonOfRequest)
+      .clickFillMyDetails()
+      .clickSubmitButton()
+      .assertResourceCreateSuccess();
+  });
+
+  it("Verify created resource", () => {
```

Comment:
```suggestion
  it("Verify created resource", () => {
```

keep creation and verification as a single test,

Reply:
done